pkg/e2e/autoscaler/autoscaler.go

```diff
@@ -77,8 +77,8 @@ func newWorkLoad() *batchv1.Job {
 				},
 			},
 			BackoffLimit: pointer.Int32Ptr(4),
-			Completions:  pointer.Int32Ptr(50),
-			Parallelism:  pointer.Int32Ptr(50),
+			Completions:  pointer.Int32Ptr(100),
```

Comment:
As @frobware suggested, we need to dynamically estimate size of the workload based on available node resources.

Reply:
adding a TODO would be nice